<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteSortOptionsWidget</class>
 <widget class="QWidget" name="NoteSortOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Sort by:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="sortRoleComboBox">
     <item>
      <property name="text">
       <string>Creation time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last modification time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Title</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Order:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="sortOrderComboBox">
     <item>
      <property name="text">
       <string>Ascending</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Descending</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="aboutNotes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/images.qrc">
       <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
